src/context/app_context.rs

```diff
@@ -20,6 +21,11 @@ pub enum QuitType {
     ChooseFiles,
 }
 
+#[derive(Clone, Debug, PartialEq)]
```

Comment:
Maybe this should be moved in it's own module at some time. But it's so small,  I thought that could wait....